sound/soc/sof/imx/imx8.c

```diff
@@ -21,6 +21,7 @@
 #include <linux/firmware/imx/svc/misc.h>
 #include <dt-bindings/firmware/imx/rsrc.h>
 #include "../ops.h"
+#include "../sof-audio.h"
```

Comment:
The commit message isn't self explanatory. I had to look at the rest of the code to figure out that you are now passing a snd_sof_pcm_stream pointer, and that structure contains a snd_pcm_substream pointer.
So it's not a functionality change, just using a higher level structure to introduce flexibility for compressed substreams

Reply:
Exactly! As you said, I think to enhance the commit messag.e

Reply:
@plbossart fixed.